src/_pyinstaller_hooks_contrib/tests/test_libraries.py

```diff
@@ -1299,6 +1300,47 @@ def test_pyqtgraph_colormap(pyi_builder):
     )
 
 
+@importorskip('pyqtgraph')
+@importorskip('PyQt5')
+def test_pyqtgraph_remote_graphics_view(pyi_builder):
+    pyi_builder.test_source(
+        """
+        import sys
+        import os
+        import signal
+
+        from PyQt5 import QtCore, QtWidgets
+        import pyqtgraph
+
+        # Multiprocessing is used internally by pyqtgraph.multiprocess
+        import multiprocessing
+        multiprocessing.freeze_support()
+
+        # pyqtgraph.multiprocess also uses a subprocess.Popen() to spawn its
+        # sub-process, so we need to restore _MEIPASS2 to prevent the executable
+        # to unpacking itself again in the subprocess.
+        os.environ['_MEIPASS2'] = sys._MEIPASS
```

Comment:
Any reason why this can't go in a runtime hook?

Reply:
Yeah, it would potentially cause issues when trying to run a different PyInstaller-frozen executable from the program, which is the main reason why the environment variable is cleared in the first place.

I think the only way to handle this automatically in the safe way is if we manage to patch `subprocess.Popen` to detect that it is trying to run `sys.executable`.

Reply:
> Yeah, it would potentially cause issues when trying to run a different PyInstaller-frozen executable from the program, which is the main reason why the environment variable is cleared in the first place.

Sure, the same problem arises if the user sets the environment variable in their code, but *they* should know whether their program will also try to spawn a subprocess with a different pyinstaller-frozen executable or not.